pydis_site/apps/api/tests/test_reminders.py

```diff
@@ -154,18 +153,26 @@ def test_reminders_in_full_list(self):
         response = self.client.get(url)
 
         self.assertEqual(response.status_code, 200)
+        self.assertNotIn(self.rem_dict_two, response.json())
+
+        response = self.client.get(f"{url}?include_active=true")
         self.assertCountEqual(response.json(), [self.rem_dict_one, self.rem_dict_two])
 
     def test_filter_search(self):
         url = reverse('api:bot:reminder-list')
         response = self.client.get(f'{url}?search={self.author.name}')
+        self.assertEqual(response.status_code, 200)
+        self.assertCountEqual(response.json(), [self.rem_dict_one])
 
+    def test_filter_search_with_inactive(self):
+        url = reverse('api:bot:reminder-list')
+        response = self.client.get(f'{url}?search={self.author.name}&include_active=true')
         self.assertEqual(response.status_code, 200)
         self.assertCountEqual(response.json(), [self.rem_dict_one, self.rem_dict_two])
 
-    def test_filter_field(self):
+    def test_only_active_by_default(self):
         url = reverse('api:bot:reminder-list')
-        response = self.client.get(f'{url}?active=true')
+        response = self.client.get(f'{url}')
```

Comment:
```suggestion
        response = self.client.get(url)
```

Reply:
Thanks, sorted in https://github.com/python-discord/site/compare/cedb0f1d724f4f2b9cb99d792331338695c3f7c3..52395f86fa3dd703372c878c739a225c366da98f